assets/Main.storyboard

```diff
@@ -1,35 +1,62 @@
+                                    <color key="titleColor" white="1" alpha="1" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
+                                </state>
+                                <connections>
+                                    <action selector="generateComic:" destination="S9b-bH-BAh" eventType="touchUpInside" id="R99-x0-TcZ"/>
+                                </connections>
+                            </button>
+                            <imageView clipsSubviews="YES" userInteractionEnabled="NO" contentMode="scaleAspectFit" horizontalHuggingPriority="251" verticalHuggingPriority="251" image="water" translatesAutoresizingMaskIntoConstraints="NO" id="AQf-oT-xOe">
+                                <rect key="frame" x="0.0" y="44" width="414" height="672"/>
+                            </imageView>
                         </subviews>
-                        <color key="backgroundColor" systemColor="systemBackgroundColor" cocoaTouchSystemColor="whiteColor"/>
+                        <color key="backgroundColor" white="1" alpha="1" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
+                        <constraints>
+                            <constraint firstItem="AQf-oT-xOe" firstAttribute="top" secondItem="eSM-oR-ihb" secondAttribute="top" id="5pP-yZ-SdS"/>
+                            <constraint firstItem="AQf-oT-xOe" firstAttribute="leading" secondItem="eSM-oR-ihb" secondAttribute="leading" id="HmF-9T-JaQ"/>
+                            <constraint firstItem="eSM-oR-ihb" firstAttribute="trailing" secondItem="AQf-oT-xOe" secondAttribute="trailing" id="N2k-TL-ekF"/>
+                            <constraint firstItem="NWr-ih-gdw" firstAttribute="centerX" secondItem="eSM-oR-ihb" secondAttribute="centerX" id="Upz-Wj-y9I"/>
+                            <constraint firstItem="AQf-oT-xOe" firstAttribute="height" secondItem="V0i-pd-vXS" secondAttribute="height" multiplier="0.75" id="hMc-gO-uim"/>
+                            <constraint firstItem="NWr-ih-gdw" firstAttribute="top" secondItem="AQf-oT-xOe" secondAttribute="bottom" constant="10" id="xyf-El-U0Y"/>
+                        </constraints>
                         <viewLayoutGuide key="safeArea" id="eSM-oR-ihb"/>
                     </view>
+                    <connections>
+                        <outlet property="comicImage" destination="AQf-oT-xOe" id="Erg-IE-RfQ"/>
+                    </connections>
                 </viewController>
                 <placeholder placeholderIdentifier="IBFirstResponder" id="jiQ-GS-bvp" userLabel="First Responder" customClass="UIResponder" sceneMemberID="firstResponder"/>
             </objects>
-            <point key="canvasLocation" x="-75" y="138"/>
+            <point key="canvasLocation" x="-75.362318840579718" y="137.94642857142856"/>
         </scene>
     </scenes>
+    <resources>
+        <image name="water" width="1920" height="1188"/>
+    </resources>
 </document>
```

Comment:
Nathan and I created some of this UI programmatically as part of the "A user should be able to view a previous commic" story. 

I would merge in those changes, and go with the programmatic UI because: 
1) It's good practice doing the UI programmatically 
2) It will make merge conflicts easier to reason about.